Civi/Install/Requirements.php

```diff
@@ -307,7 +307,8 @@ public function checkMysqlVersion(array $db_config) {
       return $results;
     }
 
-    if (version_compare($info, $min) == -1) {
+    $versionDetails = mysqli_query($conn, 'SELECT version() as version')->fetch_assoc();
```

Comment:
IMHO, it makes perfect sense to use `SELECT version()` for this.

I asked Seamus what was wrong the previous lookup - he pointed out that `mysqli_get_server_info()` for MariaDB reports weird values: https://www.php.net/manual/en/mysqli.get-server-info.php#118822

So this change makes sense